sklearn/multioutput.py

```diff
@@ -37,12 +39,89 @@ def _fit_estimator(estimator, X, y, sample_weight=None):
     return estimator
 
 
+def _partial_fit_estimator(estimator, X, y, classes=None, sample_weight=None,
+                           first_time=True):
+    if first_time:
+        estimator = clone(estimator)
+    else:
+        estimator = copy.deepcopy(estimator)
```

Comment:
oh... yeah, when doing the second fit, i have already separated estimators. no clone/copy is needed anymore.

Thanks for the catch 👍